test/mocha/10-keystore.js

```diff
@@ -31,6 +35,204 @@ describe('bedrock-kms-http API', () => {
       result.should.have.property('controller');
       result.controller.should.equal(capabilityAgentId);
     });
+    it('throws error on no sequence in postKeystore validation', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f676';
+      const handle = 'testKey1';
+
+      const capabilityAgent = await CapabilityAgent
+        .fromSecret({secret, handle});
+      const kmsBaseUrl = `${bedrock.config.server.baseUri}/kms`;
+      const url = `${kmsBaseUrl}/keystores`;
+      const config = {
+        controller: capabilityAgent.id
+      };
+      const {httpsAgent} = brHttpsAgent;
+
+      let err;
+      let result;
+      try {
+        result = await axios.post(
+          url,
+          config,
+          {httpsAgent}
+        );
+      } catch(e) {
+        err = e;
+      }
+      should.exist(err);
+      should.not.exist(result);
+      err.response.data.message.should.equal(
+        'A validation error occured in the \'postKeystore\' validator.');
+    });
+    it('gets a keystore', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f676';
+      const handle = 'testKey1';
+
+      const capabilityAgent = await CapabilityAgent
+        .fromSecret({secret, handle});
+
+      const keystore = await helpers.createKeystore({capabilityAgent});
+      let err;
+      let result;
+      try {
+        result = await helpers.getKeystore({id: keystore.id});
+      } catch(e) {
+        err = e;
+      }
+      assertNoError(err);
+      should.exist(result);
+      result.should.have.property('id');
+      result.id.should.equal(keystore.id);
+    });
+    it('finds a keystore', async () => {
+      const secret = ' b07e6b31-d910-438e-9a5f-08d945a5f676';
+      const handle = 'testKey1';
+      // eslint-disable-next-line max-len
```

Comment:
since this is a preventable case, just moving the value down to the next line with an indent is preferred over eslint directives.

Reply:
This one has not been addressed.